app_remote/flaskr/static/js/app.js

```diff
@@ -0,0 +1,17 @@
+import { PiiAppClient } from './piiAppClient.js'
```

Comment:
Do we feel the file name `app.js` make sense at all? I am having a difficult time coming up with a descriptive name, all I got is `sendData.js` or `forwardTicketId.js`. New names are welcome!

Reply:
app.js is pretty standard across codebases I've seen @gabechu

Reply:
forwardTicketId.js seems like a likely choice given the content. I'm missing a lot of context here though. Also, it looks like the function is triggering at js load time, not at function trigger time. Trigger-time would allow easier testability, however again, i'm missing a lot of context here :)

Reply:
Cool! I have the file renamed to `forwardTicketId`.